### DISCUSSION TITLE
Avoid Comps bloat

### DISCUSSION THREAD

**Discussion by marcosfrm** [rhinstaller/anaconda#4691] [Q&A]
Some time ago I asked on IRC if it was possible, in Anaconda, configure DNF to use `install_weak_deps=False` and `group_package_types=mandatory` (without `default`), to avoid Comps bloat. It was not if I remember correctly. Any change? Can I override the repository's `comps.xml` somehow?

**Comment by VladimirSlavik** [comment]
@poncovka you would probably know the answer already, I think? If not, ping me back so I know I'm the one checking.

**Comment by poncovka** [comment]
You should be able to do that with a kickstart file for specific groups (https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#chapter-9-package-selection):

```
%packages --exclude-weakdeps
@some-group --nodefaults
@another-group --nodefaults
%end
```
